lib/cylc/task_pool.py

```diff
@@ -237,35 +236,23 @@ def add_to_runahead_pool(self, itask):
             self.log.warning(
                 itask.identity +
                 ' cannot be added to pool: task ID already exists')
-            del itask
```

Comment:
They don't do anything! The `del` statement would remove the symbol `itask` from the local scope (and hence reduces the reference count by 1 to the data it references), but given that:

1. `itask` is passed into the method as an argument, the actual data is likely to be still referenced by the caller, so it is not like we can really free up the memory here.
2. Both statements are followed by a `return` statement, so the symbol will be out of scope any way.